src/test/java/teammates/test/pageobjects/InstructorCourseStudentDetailsEditPage.java

```diff
@@ -73,7 +77,11 @@ public InstructorCourseStudentDetailsEditPage submitUnsuccessfully(String studen
         if (comments != null) {
             fillTextBox(commentsTextbox, comments);
         }
-        submitButton.click();
+        if(teamName != null){
+            clickAndConfirm(submitButton);
+        }else{
+            submitButton.click();
```

Comment:
To reduce code duplication, can you move the similar part of code in both methods related to filling in the form fields into a new method?